static/js/tools/statsvar_menu.tsx

```diff
@@ -31,6 +31,8 @@ class Node extends Component<NodePropType, NodeStateType> {
     this._handleCheckboxClick = this._handleCheckboxClick.bind(this);
     this._handleExpandClick = this._handleExpandClick.bind(this);
     this._handleHashChange = this._handleHashChange.bind(this);
+    this.expandable = this.expandable.bind(this);
+    this.validSV = this.validSV.bind(this);
```

Comment:
might call it isValidStatsVar().  also can you standardize all the stats var naming to be statsVar.

Reply:
Done.